doc/source/turn-off.rst

```diff
@@ -70,10 +73,36 @@ Google Cloud Platform
    These might take several minutes to clear up, but they shouldn't have anything
    related to your JupyterHub cluster after you have deleted the cluster.
 
+Microsoft Azure AKS
+-------------------
+
+1. Perform the steps in :ref:`delete-namespace`. These cloud provider agnostic steps will
+   delete the helm chart and delete the hub's namespace. This must be done before proceeding.
+
+2. Delete your resource group. You can list your active resource groups with
+   the following command
+
+   .. code-block:: bash
+
+      az aks list --output table
```

Comment:
Should this be `az group list`?

Reply:
We should also mention that this is the resource group you created, and a note of danger if you delete a different resource group